src/test/java/school/cesar/eta/unit/PersonTest.java

```diff
@@ -1,62 +1,137 @@
 package school.cesar.eta.unit;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
 import static org.junit.jupiter.api.Assertions.fail;
 
 public class PersonTest {
     @Test
     public void getName_firstNameJonLastNameSnow_jonSnow() {
-        fail();
+        Person person = new Person();
+        person.setName("Jon");
+        person.setLastName("Snow");
+        String expected = "Jon Snow";
+        String actual = person.getName();
+        Assertions.assertEquals(expected,actual);
     }
 
     @Test
     public void getName_firstNameJonNoLastName_jon() {
-        fail();
+        Person person = new Person();
+        person.setName("Jon");
+        String expected = "Jon";
+        String actual = person.getName();
+        Assertions.assertEquals(expected,actual);
     }
 
     @Test
     public void getName_noFirstNameLastNameSnow_snow() {
-        fail();
+        Person person = new Person();
+        String expected = "Snow";
+        person.setLastName("Snow");
+        String actual = person.getName();
+        Assertions.assertEquals(expected,actual);
     }
 
     @Test
     public void getName_noFirstNameNorLastName_throwsException() {
-        fail();
+        Person person = new Person();
+        String expected = "Name must be filled";
+        Exception exception = Assertions.assertThrows(RuntimeException.class, () -> person.getName(), "Name must be filled");
+        Assertions.assertEquals(expected, exception.getMessage());
     }
 
+
     @Test
     public void isBirthdayToday_differentMonthAndDay_false() {
-        fail();
+        Person person = new Person();
+        LocalDate date = LocalDate.parse("2019-08-12");
+        person.setBirthday(date);
+        Assertions.assertFalse(person.isBirthdayToday());
```

Comment:
O método `isBirthdayToday()` chama o `getNow()`, que chama `LocalDate.now()` e pega do sistema a data no momento em que é executado. Sendo assim, isso provocará intermitências no resultado do teste a depender de quando ele é executado.

**Evite basear seus testes em variáveis do sistema!** 

Para isso, crie um [stub](https://spring.io/blog/2007/01/15/unit-testing-with-stubs-and-mocks) para fixar a data retornada pelo método `getNow()`.